src/ServantElm.hs

```diff
@@ -250,6 +281,24 @@ mkRequest request =
         Nothing ->
           "Http.emptyBody"
 
+    headers =
+      if null headerList
+        then "[]"
+        else
+          brackets $
+            concatWith
+              (surround (comma <> space))
```

Comment:
You're using `brackets . concatWith (surround (comma <> space))` in two places, is there some nice name we can give to this function and factor it out?